src/Neo/Hardfork.cs

```diff
@@ -15,6 +15,7 @@ public enum Hardfork : byte
     {
         HF_Aspidochelone,
         HF_Basilisk,
-        HF_Cockatrice
+        HF_Cockatrice,
+        HF_Domovoi //  from https://github.com/neo-project/neo/pull/3290/files
```

Comment:
Remove it, we need to merge https://github.com/neo-project/neo/pull/3290 firstly and then rebase this PR onto master.

Reply:
i will update it if that one is merged first, otherwise this pr will be the one name the hardfork, no problem at all.

Reply:
@AnnaShaleva don't introduce dependency here imo

Reply:
> don't introduce dependency here imo

I just don't like the code duplication between PRs. #3290 is ready for merge and should be included into this release as long as this PR.

Reply:
> > don't introduce dependency here imo
> 
> I just don't like the code duplication between PRs. #3290 is ready for merge and should be included into this release as long as this PR.

Duplicate is because we need to apply a name for the hf, next time maybe you can create a pr to add  a hf before fixing anything.